ui/lib/core/addon/components/form-field.hbs

```diff
@@ -39,7 +39,7 @@
             </G.CheckboxField>
           {{/each}}
           {{#if this.validationError}}
-            <G.Error data-test-field-validation={{this.valuePath}}>{{this.validationError}}</G.Error>
+            <G.Error data-test-form-field-validation-error={{this.valuePath}}>{{this.validationError}}</G.Error>
```

Comment:
Nice job on this one, let's add this above the validation-warning selector you made [here](https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/general-selectors.ts#L57).